app/assets/stylesheets/edit_profile_style.css.scss

```diff
@@ -0,0 +1,94 @@
+@import url('https://fonts.googleapis.com/css?family=Roboto');
+
+body {
```

Comment:
Soll wirklich das Body in jeden Bereich anders aussehen? Auch die ganzen Content Klassen wiederholen sich doch nicht, oder?